tests/test_rtstruct_builder.py

```diff
@@ -164,6 +165,36 @@ def test_contour_data_sizes(new_rtstruct: RTStruct):
     # Then using approximation leads to less data within the contour data
     assert get_data_len_by_index(new_rtstruct, 0) < get_data_len_by_index(new_rtstruct, 1)
 
+def test_load_sorted_series(oriented_series_path):
+    series_data = image_helper.load_sorted_image_series(oriented_series_path)
+
+    _, _, slice_direction = image_helper.get_slice_directions(series_data[0])
+    np.testing.assert_allclose(np.linalg.norm(slice_direction), 1, rtol=1e-5, atol=0)
+
+    slice_spacing = image_helper.get_spacing_between_slices(series_data)
+    first_slice_position = np.array(series_data[0].ImagePositionPatient)
+
+    for i, series_slice in enumerate(series_data):
+        position = np.array(series_slice.ImagePositionPatient)
+        expected_position = first_slice_position + i * slice_spacing * slice_direction
+        np.testing.assert_allclose(position, expected_position, rtol=1e-5, atol=0)
+
+def test_nonstandard_image_orientation(oriented_rtstruct: RTStruct):
+    mask = get_empty_mask(oriented_rtstruct)
+    mask[10:70, 5:15, 1] = 1
+    mask[60:70, 5:40, 1] = 1
+
+    IOU_threshold = 0.95 # Expected accuracy
+    run_mask_iou_test(oriented_rtstruct, mask, IOU_threshold)
+
+def test_one_slice_image(one_slice_rtstruct: RTStruct):
+    mask = get_empty_mask(one_slice_rtstruct)
+    mask[10:70, 5:15, 0] = 1
+    mask[60:70, 5:40, 0] = 1
+
+    IOU_threshold = 0.95 # Expected accuracy
```

Comment:
You mentioned some of the test IOUs went to 1.0. Can we expect an accuracy of 1.0 here as well?